ringvax/__init__.py

```diff
@@ -6,6 +6,19 @@
 
 
 class Simulation:
+    PROPERTIES = [
```

Comment:
Could we do this? https://github.com/CDCgov/cfa-ring-vax-widget/pull/45#discussion_r1892471102

Reply:
Making this a type-dict wouldn't change the behavior of, e.g., `for x in self.PROPERTIES` or `if x in self.PROPERTIES`

Reply:
It would contradict my "make this a set" but we could always have both